core/senders/qq_dispatcher.py

```
@@ -279,100 +367,95 @@ async def dispatch_processed_batches_to_targets(
+                    if chunk_failed and consecutive_failures >= fail_fast_limit:
+                        remaining = sum(
+                            len(batch_data["nodes_data"])
+                            for chunk in batch_chunks[chunk_idx:]
+                            for batch_data in chunk
+                            if target_session
+                            not in target_successes.get(
+                                batch_data["batch_index"], set()
+                            )
+                        )
+                        logger.warning(
+                            f"[QQSender] 连续 {consecutive_failures} 块失败，"
```

Ruff (0.16.0)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)